templates/publisher/listing.html

```diff
@@ -143,179 +144,173 @@ <h2 class="p-heading--four">
             </select>
           </div>
         </div>
-
-        <div class="js-categories-category1-help-text row{% if snap_categories.categories[0] %} u-hide{% endif %}">
-          <div class="col-8 col-start-large-3">
-            <p class="p-form-help-text u-no-margin--bottom">
-              Include a category and your snap will be better ranked within the Snap Store.
-              It will also allow users to discover your snap via browsing the available categories.
-            </p>
+        <div class="col-12">
+          <div class="js-categories-category1-help-text row{% if snap_categories.categories[0] %} u-hide{% endif %}">
```

Comment:
Maybe it's just me, but I find this harder to read (and possibly more error prone when maintaining, because you need to remember to put space in the conditional. The browser wouldn't care if there is additional space at the end...

```suggestion
          <div class="js-categories-category1-help-text row {% if snap_categories.categories[0] %}u-hide{% endif %}">
```

This happens in several places.

Reply:
Updated them all - they were inconsistent throughout and I'm happy for it to be this way.

Reply:
Thx